pylintrc

```diff
@@ -160,7 +160,7 @@ additional-builtins=
 ignore-mixin-members=yes
 # List of classes names for which member attributes should not be checked
 # (useful for classes with attributes dynamically set).
-ignored-classes=SQLObject,twisted.internet.reactor,hashlib,google.appengine.api.memcache
+ignored-classes=SQLObject,twisted.internet.reactor,hashlib,google.appengine.api.memcache,torch
```

Comment:
Any reason you added this?

Reply:
It is to fix this issue: https://github.com/pytorch/pytorch/issues/701
I tried adding the separate functions that are used only, but for some reason that didn't work, so I reverted to the proposed solution of adding the full toch library. Tbh, I'm not 100% happy with this either.